src/components/SemesterAdministration/SemesterAdministration.tsx

```diff
@@ -26,6 +30,27 @@ interface PostalCard {
   email: string
 }
 
+interface Invitation {
+  first_name: string
+  last_name: string
+  is_participant: boolean
+}
+
+interface SchoolInvitation {
+  participants: Invitation[]
+  school_name: School
+}
```

Comment:
tieto interfaces by bolo idealne presunut do types. Konkretne do `src/types/api`, mozno tam dokonca nieco bude (to iste pre PostalCode)